### PR TITLE
add spec of op number

### PR DESCRIPTION
**Pull request by bchyl**

### CODE REVIEW

**Review by ChihChengLiang — APPROVED** [review]
LGTM

File: specs/opcode/43NUMBER.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Timestamp op code
```
Comment (ChihChengLiang) [inline specs/opcode/43NUMBER.md:1]:
```suggestion
# Number op code
```

**Review by ChihChengLiang — CHANGES_REQUESTED** [review]
Oh, sorry a nitpick

File: specs/opcode/43NUMBER.md (hunk 1 above)
Comment (bchyl) [reply specs/opcode/43NUMBER.md]: accept it, 3gs very much.

**Review by ChihChengLiang — APPROVED** [review]
Thanks for the fix!

**Comment by barryWhiteHat** [comment]
@bchyl thanks for your PR. Do you have some email address or IM we could use for direct communication ?

**Pull request merged**